@navikt/core/react/src/table/ExpandableRow.tsx

```diff
@@ -65,17 +65,40 @@ export const ExpandableRow: ExpandableRowType = forwardRef(
   ) => {
     const [internalOpen, setInternalOpen] = useState<boolean>(defaultOpen);
     const id = useId();
+    const expandableButtonRef = useRef<HTMLButtonElement | null>(null);
+    const expandableIconRef = useRef<SVGSVGElement | null>(null);
 
     const isOpen = open ?? internalOpen;
 
+    const expansionHandler = (e) => {
+      onOpenChange?.(!isOpen);
+      if (open === undefined) {
+        setInternalOpen((open) => !open);
+      }
+      e.stopPropagation();
+    };
+
+    const onRowClick = (e) => {
+      if (
+        e.target.nodeName === "TD" ||
```

Comment:
Selv om vi ikke har noen kodeeksempler på det (og stylingen for det virker litt "off"), så skulle jeg gjerne ønsket at vi sjekker for "TH" i tillegg til "TD", i tilfelle noen bruker radoverskrifter. Mest for å forhindret at vi glemmer dette om man tar i bruk radoverskrifter i fremtiden.